### ISSUE TITLE
Need an end-to-end driver CLI script with simple configuration

### ISSUE THREAD

**Issue by DaveStrickland** [DaveStrickland/AstroPhotography#2]
The existing v0.2.3 AstroPhotography python classes and command line scripts work well enough processing single images, but don't provide a means to process an entire observing nights worth of images into final form. I'm currently relying on some bash scripts I wrote but they're

- not python
- not particularly flexible because they lack a good way of having external configs
- in some key cases (resampling, color composites) rely on external non-python packages (see Issue #3)
- not included the AstroPhotography package itself

I need to prioritize addressing this, in particular having something where a simple and short yaml config can be used, that can also handle vagaries of where users put calibration files.

**Comment by DaveStrickland** [comment]
Temporarily added the `bash` scripts I have been using:
```bash
src/AstroPhotography/scripts/calibrate_all.sh	src/AstroPhotography/scripts/navigate_all.sh
src/AstroPhotography/scripts/composite_all.sh	src/AstroPhotography/scripts/resample_all.sh
```

**Comment by DaveStrickland** [comment]
Merge 91df03e6f07333753254f7db834d60f95bf4cf14 resolves #2
